label_studio_ml/wsgi.py

```diff
@@ -4,6 +4,16 @@
 import logging
 import logging.config
 
+try:
+    import torch.multiprocessing as mp
+    try:
+        # avoid "cannot reinit CUDA in forked process" error in loading cuda?
+        mp.set_start_method('spawn')
+    except RuntimeError:
+        pass
+except ImportError:
+    import multiprocessing as mp
+
```

Comment:
Why does this work here and not if I put the same commands into the example `_wsgi.py`?

Reply:
It actually does not work with gunicorn. The only solution is to remove `--preload`